acceptance/testdata/builder/container/cnb/buildpacks/hello_world/0.0.1/bin/build

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+echo "---> Hello World buildpack"
```

Comment:
Similar to https://github.com/buildpacks/lifecycle/pull/648/files#r669969132, it would be great if we could delete from this file anything that's not used by the tests.

Reply:
deleted most parts